package.json

```diff
@@ -106,13 +100,11 @@
     "marked": "^0.6.1",
     "mdn-polyfills": "^5.16.0",
     "memoize-one": "^5.0.2",
-    "moment": "^2.24.0",
     "node-vibrant": "^3.1.5",
     "regenerator-runtime": "^0.13.2",
     "resize-observer": "^1.0.0",
     "roboto-fontface": "^0.10.0",
     "superstruct": "^0.6.1",
-    "tslib": "^1.10.0",
```

Comment:
If we remove this, `"importHelpers": true,` should also be removed from `tsconfig`? I guess it doesn't really matter because babel does our typescript work anyway...